modules/gem5/_posts/2000-04-01-assignment2.md

```diff
@@ -123,6 +157,13 @@ Collect data from your simulation runs and use simulator statistics to answer th
 Use clear reasoning and visualization to drive your conclusions.
 You are allowed to submit your reports in **pairs** and in **PDF** format.
 
+### Step 0
+
+Before starting simulation and analysis, you should be able to identify the
+ROI of a  program.
+
+1. For the DAXPY's assembly code, identify the ROI. In your report, copy the assembly code segment corresponding to `m5_work_begin` and `m5_work_end`.  
```

Comment:
Do they know how to use `objdump`? If it's not already in the assignment, you should add that information.

Reply:
i have added new instructions to compile this program, generate its assembly code and also use objdump to see the instructions executed in the latest patchset.